### PR TITLE
fix: corrected install instructions and added getRawValue() to TypedFormGroup

### LINKED ISSUE #52: Incorrect README install instructions
In your README "Getting Started" section it says to use the command `npm install --save-dev ngx-forms-typed`, but the npm page sidebar says `npm i ngx-forms-typed`. I assume it's the latter, but you might want to get those in sync.

### PR DESCRIPTION
**Pull request by Josh015**
- add the support for `getRawValue()` on `TypedFormGroup`
- updated README with correct install instructions

Closes #52 #55

### CODE REVIEW

File: projects/forms/README.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+- functions for instantiating the `TypedFormControl`, `TypedFormGroup` and `TypedFormArray` based on a Model type
+- a helper function that enumerates all controls in a form (group)
+  - calls methods on each of them
+  - attaches their validators to a parent form (group)
+  - attaches the touched/untouched behavior to a parent form (group)
 
 ## Getting started
 
-1. `npm install --save-dev ngx-forms-typed`
+1. `npm i ngx-forms-typed`
```
Comment (gparlakov) [inline projects/forms/README.md:12]: Looks great, can we please get rid of the rest of the changes - whitespaces, etc.?

File: projects/forms/src/lib/forms-typed.ts [hunk 2]
```diff
@@ -129,6 +129,7 @@ export interface TypedFormGroup<K, C extends Controls<K> = TypedControlsIn<K>> e
   value: K;
   setControl: <T extends keyof C>(name: T extends string ? T : never, control: C[T]) => void;
   reset: (value?: ResetValue<K>, options?: FormEventOptions) => void;
+  getRawValue: () => K;
```
Comment (gparlakov) [inline projects/forms/src/lib/forms-typed.ts:132]: Great, I'd appreciate it if you added a test case and/or a use case respectively in `forms-typed.tests.ts` and in some of the components in the `src` example app.

**Review by gparlakov — COMMENTED** [review]
Looks great, thanks!

A couple of things:
 - we follow semantic-releases (Angular) commit message styles (so semantic release can figure out if a patch, minor or major version should be released
 - we need a CHANGELOG entry for this.

File: projects/forms/README.md (hunk 1 above)
Comment (Josh015) [reply projects/forms/README.md:12]: My bad. I forgot to turn off FormatOnSave.

File: projects/forms/src/lib/forms-typed.ts (hunk 2 above)
Comment (Josh015) [reply projects/forms/src/lib/forms-typed.ts:132]: For the example app I was just gonna add a `console.log()` to show that the values from `getRawValue()` are different from `value` with disabled controls. Is that sufficient?

Comment (Josh015) [reply projects/forms/src/lib/forms-typed.ts:132]: Okay, I've never seen testing in the style you're using and I'm at a loss for how to do it in a way that would be satisfactory.

**Comment by Josh015** [comment]
Here's what I'm gonna do. I'm gonna kill this PR and my fork so I can start fresh and try to follow the guidelines from the start.  I'll submit a small one for the README and then follow it up with another for `getRawValue()`. when I've got a handle on this.

**Pull request closed without merging**